src/main/java/fish/payara/extras/diagnostics/collection/collectors/LogCollector.java

```diff
@@ -121,7 +141,7 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
         public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
 
             Path relativePath = path.relativize(file);
-            if (!file.getFileName().toString().contains(".log")) {
+            if (!file.getFileName().toString().endsWith(fileExtension)) {
```

Comment:
This change stops the collection of old logs e.g. `server.log_2024-08-02T10-42-21`

Reply:
I have now changed this so it includes the old logs.